articles/cis-benchmarks.md

```diff
@@ -49,25 +49,65 @@ If either of these conditions fails, the host is considered to be failing the po
 
 All CIS policies are stored under our restricted licensed folder `ee/cis/`.
 
-How to import them to Fleet:
-```sh
-# Download policy queries from Fleet's repository 
-# macOS 13
-wget https://raw.githubusercontent.com/fleetdm/fleet/main/ee/cis/macos-13/cis-policy-queries.yml
+1. Install [yq](https://github.com/mikefarah/yq) if you you don't have it already.
+2. Run this Shell script to transform the policies into [Fleet YAML]([https://fleetdm.com/docs/configuration/yaml-files](https://fleetdm.com/docs/configuration/yaml-files#policies)):
+
+```
+#!/bin/bash
+#shellcheck disable=SC2207
+
+
+
+# convert.cis.policy.queries.yml @2024 Fleet Device Management
+# Brock Walters (brock@fleetdm.com)
```

Comment:
```suggestion
```

FYI @nonpunctual removing your email because we have a test to make sure no emails get added.